helm-chart/images/binderhub/dependencies

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+"""automatically manage requirements.txt dependencies with pip-tools
+
+This is an adjusted script from z2jh by @minrk to mount the binderhub python
+package requirements.txt file as a requirements.in file to freeze it to
+requirements.txt in helm-chart/images/binderhub.
+
+See
+
+    ./dependencies --help for commands and arguments
+
+How it works:
+
+- the image used in the helm chart installs a frozen environment from
+  requirements.txt
+- `pip-compile` is used to generate frozen `requirements.txt` from our actual
+  requirements in /requirements.txt next to setup.py for the binderhub package.
+- `pip list --outdated` is used to report available updates for packages in the
+  frozen environment
+- pip-compile etc. are run *inside the image* to ensure consistent behavior,
+  rather than running on host systems, which can vary.
+- When building the image to be used for running dependency-management commands,
+  chartpress configuration is loaded to ensure the environment is the same as
+  when chartpress builds the tagged image to be published.
+"""
+
+import json
+import os
+from functools import lru_cache
+from subprocess import check_call, check_output
+
+import click
```

Comment:
can you add `click` to `dev-requirements.txt` please

Reply:
Good catch thanks, done!